subsys/bluetooth/host/Kconfig

```diff
@@ -132,6 +132,27 @@ config BT_DRIVER_RX_HIGH_PRIO
 	int
 	default 6
 
+config BT_TX_PROCESSOR_THREAD
+	# This thread is used to send pending HCI Commands, ACL and ISO data to
+	# Controller.
+	bool
+	# This option is automatically selected for all platforms except nRF51
+	# due to limited RAM on nRF51 devices.
+	default n if !SOC_SERIES_NRF51X
```

Comment:
Isn't the negation incorrect here? I.e. you *don't* want this on nRF51? Maybe it'd be simpler to just have a single `default y if !SOC_SERIES_NRF51X` line?